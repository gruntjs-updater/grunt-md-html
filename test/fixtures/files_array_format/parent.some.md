parent.some.md
====

```javascript
var a = 10;
```