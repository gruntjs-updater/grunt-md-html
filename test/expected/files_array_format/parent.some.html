<h1 id="parent-some-md">parent.some.md</h1>
<pre><code class="lang-javascript"><span class="hljs-variable"><span class="hljs-keyword">var</span> a</span> = <span class="hljs-number">10</span>;
</code></pre>
